### ISSUE TITLE
CODE 2.1: Formatting issue *.ppt/*.pptx

### ISSUE THREAD

**Issue by victorbw** [CollaboraOnline/Docker-CODE#12]
Hey guys, just noticed that when I upload a ppt/pptx, which was originally formated/created on an MS-Office-Powerpoint-Suite (2003, 2007, or whatever), the formatting of slides goes terribly wrong.

Here some examples, lets beginn with the original in PowerPoint:
![image](https://cloud.githubusercontent.com/assets/8447071/26578868/d591229e-4531-11e7-9c6b-4199fdc460ba.png)

now the same in collabora/code:
![image](https://cloud.githubusercontent.com/assets/8447071/26578876/dfcbcde0-4531-11e7-98b5-9717ae0a8bc8.png)

Well, this is when I 'DON'T EDIT or SAVE' the document online.

But what happens if I do so? Well, then the formatting-issue gets synched as well.
Lets see what happened now - first, collabora:
![image](https://cloud.githubusercontent.com/assets/8447071/26578888/efa5c68a-4531-11e7-8a39-390e14bb972e.png)

now, after synchronisation, I have it on PowerPoint as well:
![image](https://cloud.githubusercontent.com/assets/8447071/26578893/f6a9783c-4531-11e7-9f8a-9107d107d1d8.png)

and this is where this error-window pops up:
![image](https://cloud.githubusercontent.com/assets/8447071/26578930/0d77b13c-4532-11e7-973e-3529948935d9.png)

translated in english:_Some texts, Images or objects on slides in the file "XYZ.pptx" could not be presented correcty by PowerPoint due to a damaged file._
[...]
_Lost informations can not be restored. [...]_

anyone knows about this issue?

**Comment by thebearon** [comment]
Can you check if you can reproduce this using desktop [LibreOffice](http://libreoffice.org/), and file a bug report in [Bugzilla](https://bugs.documentfoundation.org/) if that's the case? If not, please attach a sample file here.

**Comment by victorbw** [comment]
same issue, the files will get realligned somewhere it doesnt belong to.

well I was offline for a week unfortunately, sorry for that late response, but here you go with a 'cut down version' of a presentation.

you'll see on the second slide that the formatting went totally wrong (which wasnt before i've opened it with some versions of MS-suites) :
[Security.pptx](https://github.com/CollaboraOnline/Docker-CODE/files/1088157/Security.pptx)

**Comment by thebearon** [comment]
Did you have a chance to test the behavior with desktop LibreOffice?

**Comment by victorbw** [comment]
yes I had, and with LibreOffice I cant reproduce any of these formatting-issues.
